internal/loginsrv/server.go

```diff
@@ -24,7 +24,155 @@ func (srv *Server) ServeHTTP(w http.ResponseWriter, req *http.Request) {
 	case <-ctx.Done():
 		// Client disconnect. Nothing to do.
 	case srv.TokenChan <- token:
+		htmlDoc := `<html>
```

Comment:
nit: would probably be better to define this as a constant outside of the select control-flow